player/lua/stats.lua

```diff
@@ -1525,7 +1525,7 @@ local function print_page(page, after_scroll)
     end
 end
 
-local function update_scale(value)
+local function update_scale(height)
```

Comment:
I don't think this is a good rename.

You can't `update_scale(height)`.

You either set scale to a factor or value, or `update_height` with a new height value.

(I didn't check what it does exactly, so I don't know how that function should be called)

Reply:
Renamed to `osd_height`.